infra/templates/az-isolated-service-single-region/app.tf

```diff
@@ -33,3 +33,76 @@ module "keyvault" {
     "azurerm" = "azurerm.app_dev"
   }
 }
+
+module "container_registry" {
+  source                  = "../../modules/providers/azure/container-registry"
+  container_registry_name = local.acr_name
+  resource_group_name     = azurerm_resource_group.app_rg.name
+  // Note: this is requird until App Services and ACR work over MSI. See the design document for more details.
+  container_registry_admin_enabled = true
+  // Note: only premium ACRs allow configuration of network access restrictions
+  container_registry_sku = "Premium"
```

Comment:
Why are we hardcoding `Premium` as the sku?

Reply:
This is described in the comment above but only premium SKUs allow us to configure network ACLs. Any other value for this will result in an error from Azure when attempting to configure the VNET integration.